node/src/components/consensus/era_supervisor.rs

```diff
@@ -334,8 +341,9 @@ where
         effect_builder: EffectBuilder<REv>,
         rng: &mut NodeRng,
         switch_blocks: &[BlockHeader],
+        merkle_tree_hash_activatio: EraId,
```

Comment:
```suggestion
        merkle_tree_hash_activation: EraId,
```

But I think that can be removed anyway and taken from the chainspec:

```suggestion
```

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/2531/commits/70cc245e4930d1ed0e81d79d0d212d3a73bbe8a0